src/Build/Logging/BinaryLogger/BinaryLogger.cs

```diff
@@ -135,6 +135,12 @@ public enum ProjectImportsCollectionMode
         /// </summary>
         public string Parameters { get; set; }
 
+        /// <summary>
+        /// Optional expander of wildcard(s) within the path parameter of a binlog <see cref="Parameters"/>.
+        /// See <see cref="IBinlogPathParameterExpander"/> for more details.
+        /// </summary>
+        public IBinlogPathParameterExpander PathParameterExpander { private get; set; } = new BinlogPathParameterExpander();
```

Comment:
I thought you said simplistic ;) Just use a `Func<string>` here (or `Func<string, string>`) and keep the default value null.

If there are currently no callers setting it from outside, we should make it internal.

Reply:
Absolutely.. it was just my excessive love for interfaces. And then I needed to expose it due to easier mocking.
Lm flip to func - which will solve both of those